src/MIDebugEngine/AD7.Impl/AD7Disassembly.cs

```diff
@@ -89,6 +89,20 @@ private DisassemblyData FetchBadInstruction(enum_DISASSEMBLY_STREAM_FIELDS dwFie
                 dis.dwFields |= enum_DISASSEMBLY_STREAM_FIELDS.DSF_OPCODE;
                 dis.bstrOpcode = "??";
             }
+
+            if ((dwFields & enum_DISASSEMBLY_STREAM_FIELDS.DSF_DOCUMENTURL) != 0)
+            {
+                dis.dwFields |= enum_DISASSEMBLY_STREAM_FIELDS.DSF_DOCUMENTURL;
+                dis.bstrDocumentUrl = string.Empty;
+            }
```

Comment:
I don't think you need this. It should be fine to leave the documentUrl empty + no DSF_DOCUMENTURL for instructions without a source file.